Gaijinizer patch/Map389.txt

```diff
@@ -255,465 +301,588 @@ It's one of my favorite name over all this time.#DL
-
+> CONTEXT: Map389/events/3/pages/0/162/Dialogue/enemy29
+> CONTEXT: Map389/events/11/pages/0/162/Dialogue/enemy29
+Continuing, I'll answer your earlier question.
+Like you said, \\C[2]I'm not a resident of this world.
+I flew here from another world just as you did.#Ron
 > END STRING
 
 > BEGIN STRING
 誰の差し金か知らないけど
 
 キミは『本来の世界』から隔離された空間から
 別世界への『本来ありえない介入』を続けている
-> CONTEXT: Map389/events/3/pages/0/167/Dialogue/enemy29 < UNTRANSLATED
-> CONTEXT: Map389/events/11/pages/0/167/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map389/events/3/pages/0/167/Dialogue/enemy29
+> CONTEXT: Map389/events/11/pages/0/167/Dialogue/enemy29
+I don't know who put you up to it, but you've
+traveled to another world from a space separate
+from your "original world" to continue
+"interventions that should not be possible".#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 常識的に考えて、そんな現実改変みたいな行為が
 キミだけに許されると思わないよね？
 
 そもそも『介入』自体がイレギュラーな行為なのに…
-> CONTEXT: Map389/events/3/pages/0/172/Dialogue/enemy29 < UNTRANSLATED
-> CONTEXT: Map389/events/11/pages/0/172/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map389/events/3/pages/0/172/Dialogue/enemy29
+> CONTEXT: Map389/events/11/pages/0/172/Dialogue/enemy29
+If you apply a little common sense, why should
+you be the only one allowed to alter reality like
+this? Especially considering how irregular the
+interventions themselves are.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 さらに、冥府の間の連中が渡した『悪魔全書』も相まって
 キミは世の摂理を超えた知識や経験を持ち始めた……
 
 ボクからすれば、キミはもう立派な\\C[10]『特異点』\\C[0]さ
-> CONTEXT: Map389/events/3/pages/0/177/Dialogue/enemy29 < UNTRANSLATED
-> CONTEXT: Map389/events/11/pages/0/177/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map389/events/3/pages/0/177/Dialogue/enemy29
+> CONTEXT: Map389/events/11/pages/0/177/Dialogue/enemy29
+And coupled with the "compendium" you got from
+those guys at Hell's Gate, your knowledge and
+experience is exceeding the world's divine
+providence.
+From my point of view, you're already quite the
+fabulous \\C[10]"Singularity"\\C[0].#RonIffy
 > END STRING
 
 > BEGIN STRING
 と…『特異点』……？
-> CONTEXT: Map389/events/3/pages/0/182/Dialogue/kazuya < UNTRANSLATED
-> CONTEXT: Map389/events/11/pages/0/182/Dialogue/kazuya < UNTRANSLATED
-Si.."Singularity"...?#Triketos
+> CONTEXT: Map389/events/3/pages/0/182/Dialogue/kazuya
+> CONTEXT: Map389/events/11/pages/0/182/Dialogue/kazuya
+Si-"Singularity"...?#Triketos
 > END STRING
 
 > BEGIN STRING
 世界を一片の慈悲なく壊すもの…………
 あるいは、危うく世界の平穏を保つもの…
 
 強大な力は、１つ存在するだけでも危険なの
-> CONTEXT: Map389/events/3/pages/0/184/Dialogue/enemy29 < UNTRANSLATED
-> CONTEXT: Map389/events/11/pages/0/184/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map389/events/3/pages/0/184/Dialogue/enemy29
+> CONTEXT: Map389/events/11/pages/0/184/Dialogue/enemy29
+Something that can either mercilessly destroy the
+world, or barely maintain world peace. It is
+mighty and dangerous power that can only exist
+in a single entity.#RonIffy
 > END STRING
```

Comment:
Some rewording for effect and flow:
```
Someone who can mercilessly destroy a world...
Or can almost create world peace by themselves.
```

Not sure about how I want the last line yet.